### PR TITLE
Fix route not found when called from other namespace

### PR DESCRIPTION
**Pull request by mtomov**
Similar to the colleague below @arthurtalkgoal , another namespace fix

### CODE REVIEW

**Comment by TeatroIO** [comment]
I've prepared a stage to preview changes. <a href='http://mtomov-add-spree-namespace-to-route.spree-spree-9ed724b77adec04c49c2.ttrcloud.com' target='_blank'>Open stage</a> or <a href='http://mtomov-add-spree-namespace-to-route.spree-spree-9ed724b77adec04c49c2.ttrcloud.com/teatro.log' target='_blank'>view logs</a>.

File: frontend/app/helpers/spree/frontend_helper.rb [hunk 1]
```diff
@@ -10,7 +10,7 @@ def breadcrumbs(taxon, separator="&nbsp;")
...
       if taxon
-        crumbs << content_tag(:li, content_tag(:span, link_to(Spree.t(:products), products_path, itemprop: "url") + separator, itemprop: "title"), itemscope: "itemscope", itemtype: "http://data-vocabulary.org/Breadcrumb")
+        crumbs << content_tag(:li, content_tag(:span, link_to(Spree.t(:products), spree.products_path, itemprop: "url") + separator, itemprop: "title"), itemscope: "itemscope", itemtype: "http://data-vocabulary.org/Breadcrumb")
         crumbs << taxon.ancestors.collect { |ancestor| content_tag(:li, content_tag(:span, link_to(ancestor.name , seo_url(ancestor), itemprop: "url") + separator, itemprop: "title"), itemscope: "itemscope", itemtype: "http://data-vocabulary.org/Breadcrumb") } unless taxon.ancestors.empty?
         crumbs << content_tag(:li, content_tag(:span, link_to(taxon.name , seo_url(taxon), itemprop: "url"), itemprop: "title"), class: 'active', itemscope: "itemscope", itemtype: "http://data-vocabulary.org/Breadcrumb")
```
Comment (JDutil) [inline frontend/app/helpers/spree/frontend_helper.rb]: Wouldn't this seo_url and above need it too?

Comment (mtomov) [inline frontend/app/helpers/spree/frontend_helper.rb]: Right, I was thinking the same, but it turned out that the seo_url is coming from the [`BaseHelper`](https://github.com/spree/spree/blob/8da7cc4fe2ac062de7e37bfdcf49bef9f57d2ac5/core/app/helpers/spree/base_helper.rb#L76)

**Comment by peterberkenbosch** [comment]
looks good to me, the seo_url is for taxons, not products. :+1: The failing spec seems unrelated.

**Comment by JDutil** [comment]
Hmm we should probably consider moving that seo_url helper to the routes file.

**Pull request closed without merging**